net/nginx/files/nginx.init

```diff
@@ -75,6 +75,54 @@ nginx_init() {
 	logger -t "nginx_init" -p "daemon.info" "using ${CONF} (the test is ok)"
 }
 
+add_mdns() {
+        local cfg="$1"
+        local port enabled service
+
+        config_get enabled "$cfg" enabled
+        config_get port "$cfg" listen
+
+        port=$(echo "$port" | head -n1 | awk '{print $1;}')
+
+
+        if [ "${enabled}" != "0" ] && [ "${enabled}" != "false" ] && [ -n "${port}" ]; then
```

Comment:
Any reason you chose  `"${...}"` ? I see it used in a few other places in the file, but keep it to `"$..."` or `${...}`.

Also, indent inside the if block.

Apply tabs consistently to indent inside this function block. Line up this function indent with the function above.

Reply:
Hello @systemcrash,
- ${} was me being "extra careful" in the in the test operator :)
- I like to have consistent indentation neither vim nor github show the indentation, and vim does not have great support for indentation in shell scripts (it is good for C), so I missed that. I will try to use VisualStudio editor for this in future.